### PR TITLE
#23 Troubleshooting/Monitor/Equally named metrics.

### LINKED ISSUE #23: Troubleshooting part + Monitor: equally named metrics
For general know problems or issues that can not be solved within reasonable time, let's introduce a new top-level chapter "Troubleshooting". 

And the first chapter "Monitor/Equally named metrics" describing https://github.com/datacleaner/DataCleaner/issues/1440 problem and its workaround.

### PR DESCRIPTION
**Pull request by jhorcicka**
Resolves #23. 
- New top-level chapter "Troubleshooting". 
- Section "DataCleaner Monitor". 
- Issue "equally named metrics" while creating a timeline for a job with multiple same-type analyzers. 
- Workaround.

### CODE REVIEW

File: src/docbkx/chapter91-monitor.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+	<title>DataCleaner monitor</title>
+
+	<chapterinfo>
+		<abstract>
+			<para>
+				Following sections describe known issues you may encounter while using
+				DataCleaner Monitor. You will also find a workaround or tips how to
+				fix the problem when such a solution exists.
+			</para>
```
Comment (ClaudiaPHI) [inline src/docbkx/chapter91-monitor.xml:14]: Can we use plural here ? Such as "workarounds", "problems", "solutions".  After "tips" word "for" is missing

File: src/docbkx/chapter91-monitor.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		</abstract>
+	</chapterinfo>
+
+	<section id="equally_named_metrics">
+		<title>Equally named metrics</title>
```
Comment (ClaudiaPHI) [inline src/docbkx/chapter91-monitor.xml]: Can we change to " Metrics with identical names" ?

**Comment by ClaudiaPHI** [comment]
Should we move this section to the timelines section: 

https://datacleaner.org/resources/docs/5.1/html/ch03s05.html ? 

image

@LosD , @jhorcicka

File: src/docbkx/chapter91-monitor.xml (hunk 1 above)
Comment (LosD) [reply src/docbkx/chapter91-monitor.xml:14]: I'd write "You can also find workarounds and tips on how to" etc.

File: src/docbkx/chapter91-monitor.xml (hunk 2 above)
Comment (LosD) [reply src/docbkx/chapter91-monitor.xml]: I think I like it better as is.

**Comment by LosD** [comment]
If it fits, we could. But the overall idea with introducing a new section was to have a place to go to when running into trouble.

... Of course that doesn't make sense unless we find more stuff to put in there, but I think we will.

**Comment by LosD** [comment]
Looking overall at the style, I think I'd reorder it a bit, so that the first paragraph is the actual trouble the user is having, like "Some of my metrics are missing", and then I'd be more consise. Less/smaller images, and keeping the text as short as possible.

Maybe even use the question itself as the "title", so that what people see first, and so that it's in the section overview. If/when more questions are in the section, it becomes important for the user to be able to easily scan for the issue s/he's seeing.

**Comment by jhorcicka** [comment]
@LosD, @ClaudiaPHI: The content was updated according to your suggestions. I only wanted to keep "equally named metrics" in the title. Now it is "Are some (equally named) metrics missing?". Without that I would look into this chapter whenever I have something metrics-related missing in my graph which would probably be a different issue.

**Comment by ClaudiaPHI** [comment]
LGTM!

**Comment by LosD** [comment]
I don't like having the equally named thing in the title/question.

The issue is that some metrics are missing, the reason is that they have the same name, the solution is to rename them.

Those three parts should really be in their own parts.

**Comment by LosD** [comment]
You _should_ look into this chapter when ever you have metrics missing. Other causes for the same issue should go here as well

**Comment by LosD** [comment]
The problem itself should be simple and easily recognized, much like a troubleshooting guide in physical manuals:
image

(The example is not exactly a nice layout, but it shows the idea)

**Comment by jhorcicka** [comment]
Updated, @LosD.

**Comment by LosD** [comment]
LGTM!

**Pull request merged**